### ISSUE TITLE
[Dataset Title/Name]: Global seamless and high-resolution temperature dataset (GSHTD), 2001–2020

### ISSUE THREAD

**Issue by TC25** [samapriya/awesome-gee-community-datasets#210]
### Contact Details

tc.chakraborty25@gmail.com

### Dataset description

Global 1 km monthly air temperature, all-sky land surface temperature, and clear-sky land surface temperature for day and night from 2001 to 2020 from this paper: https://doi.org/10.1016/j.rse.2022.113422

All the geotiffs are here: https://drive.google.com/drive/u/3/folders/1jvotZ91vV840UKCwxumfa35I5LTqGNXp

### Earth Engine Snippet if dataset already in GEE

for example
```js
var asset_name = ee.ImageCollection("path to your collection")
```
Sample Code: Add a sample code maybe just adding your datasets in the code editor

### Enter license information

Enter License information ex: APACHE 2.0, CC-BY-4.0

### Keywords

air temperature, all-sky, land surface temperature, clear-sky

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by samapriya** [comment]
Hi @TC25 
Can you find out if they have license information? I cannot add these without knowing license & redistribution rights

**Comment by TC25** [comment]
Hi, @samapriya!

I don't think they paid attention to the license issue when publishing this. I did reach out to the lead author and they agree to the CC-By attribution. Would that work? I will forward the email.

**Comment by samapriya** [comment]
This has been completed and will be included in the next release. Thanks for following up on the license @TC25

**Comment by samapriya** [comment]
Hey @TC25 
Can you specify the scaling and offset for this data seems the range is really high for these values so they are not a direct kelving to celcius conversion I presume

**Comment by samapriya** [comment]
Resolved to 0.02 as recommended closing this issue again
